__test__/response.spec.ts

```diff
@@ -197,20 +197,54 @@ describe('Response', () => {
     }).toThrow();
   });
 
-  it('should fail if header is invalid', () => {
+  it('should fail if header value is invalid', () => {
     const responseMeta: ResponseMeta = {
       status: 200,
       header: {
         'content-type': 'application/json',
         version: [1, 2],
       },
     };
-    expect(() => {
+
+    let thrownError: ResponseValidationError;
+
+    // test validateResponseByPath
+    try {
       chowchow.validateResponseByPath('/header', 'get', responseMeta);
-    }).toThrow();
-    expect(() => {
+      fail('An exception should have been thrown');
+    } catch (error) {
+      thrownError = error;
+    }
+    expect(thrownError.toJSON().suggestions[0].error).toMatch(
+      new RegExp('version: type should be string')
+    );
+
+    // test validateResponseByOperationId
+    try {
       chowchow.validateResponseByOperationId('getHeader', responseMeta);
-    }).toThrow();
+      fail('An exception should have been thrown');
+    } catch (error) {
+      thrownError = error;
+    }
+    expect(thrownError.toJSON().suggestions[0].error).toMatch(
+      new RegExp('version: type should be string')
+    );
+  });
+
+  it('should succeed if header case is different than spec', () => {
+    const responseMeta: ResponseMeta = {
+      status: 200,
+      header: {
+        version: '1',
```

Comment:
Would it still work if now that we return a header in upper case? Like `Version : '1'`

Reply:
Like I mentioned in another comment, I added a test for this. It's not sending both `version` and `Version`.

Reply:
Thanks, it looks good.